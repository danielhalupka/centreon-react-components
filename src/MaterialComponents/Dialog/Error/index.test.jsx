/* eslint-disable no-undef */

import React from 'react';
import { render } from '@testing-library/react';
import ErrorDialog from '.';

describe('ErrorDialog', () => {
  it('renders', async () => {
    const { container, findByText } = render(
      <ErrorDialog
        open
        title="Error"
        text="Something unexpected happened..."
        confirmLabel="Close"
        onClose={() => {}}
      />,
    );

    await findByText('Error');

    // TODO this provides an empty Snapshot. This is probably due to the Fade animation not handled by react-testing-library.
    expect(container.firstChild).toMatchSnapshot();
  });
});
