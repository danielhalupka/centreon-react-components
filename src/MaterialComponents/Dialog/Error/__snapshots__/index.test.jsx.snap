// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDialog renders 1`] = `null`;
